hazelcast-client/src/test/java/com/hazelcast/client/map/ClientMapBasicTest.java

```diff
@@ -72,765 +82,545 @@ public static void destroy() {
 
     @Test
     public void testClientGetMap() {
-        assertNotNull( client.getMap(randomString()) );
+        assertNotNull(client.getMap(randomString()));
     }
 
     @Test
     public void testGetName() {
-        String mapName = randomString();
-        final IMap map = client.getMap(mapName);
-        assertEquals(mapName, map.getName());
+        assertEquals(randomMapName, randomStringMap.getName());
     }
 
     @Test
     public void testSize_whenEmpty() {
-        final IMap map = client.getMap(randomString());
-        assertEquals(0, map.size());
+        assertEquals(0, randomStringMap.size());
     }
 
     @Test
     public void testSize() {
-        final IMap map = client.getMap(randomString());
-        map.put("key", "val");
-        assertEquals(1, map.size());
+        randomStringMap.put(KEY, NEW_VALUE);
+        assertEquals(1, randomStringMap.size());
     }
 
     @Test
     public void testSize_withMultiKeyPuts() {
-        final IMap map = client.getMap(randomString());
-        final Object key = "Key";
-        final Object oldValue = "oldValue";
-        final Object newValue = "Val";
-
-        map.put(key, oldValue);
-        map.put(key, newValue);
+        randomStringMap.put(KEY, OLD_VALUE);
+        randomStringMap.put(KEY, NEW_VALUE);
 
-        assertEquals(1, map.size());
+        assertEquals(1, randomStringMap.size());
     }
 
     @Test
     public void testIsEmpty_whenEmpty() {
-        final IMap map = client.getMap(randomString());
-        assertTrue(map.isEmpty());
+        assertTrue(randomStringMap.isEmpty());
     }
 
     @Test
     public void testIsEmpty_whenNotEmpty() {
-        final IMap map = client.getMap(randomString());
-        map.put("key", "val");
-        assertFalse(map.isEmpty());
+        randomStringMap.put(KEY, NEW_VALUE);
+        assertFalse(randomStringMap.isEmpty());
     }
 
     @Test
     public void testIsEmpty_afterPutRemove() {
-        final IMap map = client.getMap(randomString());
-        final Object key = "key";
-        map.put(key, "val");
-        map.remove(key);
-        assertTrue(map.isEmpty());
+        randomStringMap.put(KEY, NEW_VALUE);
```

Comment:
For me it is the absolute opposite. Especially 'K' and 'k' can be hard to differ. This may be intended (check for different keys), but if you don't use a constant (which is easy to spot because of the UPPERCASE and unique color), you have to parse all strings while reading the tests. You can also easily make a typo in each hand-written string. For constants you have auto-completion.